docs/products/clickhouse/howto/integrate-pg.rst

```diff
@@ -0,0 +1,106 @@
+
+.. note::
+
+    Aiven for ClickHouse service integrations are available for Startup plans and higher.
+
+Prerequisites
+-------------
+
+You will need:
+
+* Aiven for ClickHouse service
+* Aiven for PostgreSQL service or a self-hosted PostgreSQL service
+* If you use the self-hosted PostgreSQL service, an external PostgreSQL endpoint should be configured in **Service Integrations**
+* These instructions assume that you have at least one table in your PostgreSQL service
+
+Variables
+-------------
+
+The following variables will be used later in the code snippets:
+
+============================     ==========================================================================================================
+Variable                         Description
+============================     ==========================================================================================================
+``CLICKHOUSE_SERVICE_NAME``      Name of your Aiven for ClickHouse service.
+``PG_SERVICE_NAME``              Name of the PostgreSQL service you use for the integration.
+``PG_DATABASE``                  Name of PostgreSQL database you're integrating.
+``PG_SCHEMA``                    Name of PostgreSQL schema.
+``PG_TABLE``                     Name of the PostgreSQL table you use for the integration.
+============================     ==========================================================================================================
+
+Create an integration
+----------------------
+
+You can create an integration with the help of :ref:`Aiven CLI <avn_service_integration_create>`, or through the `Aiven console <https://console.aiven.io/>`_ by following these steps:
+
+1. Log in to the `Aiven web console <https://console.aiven.io/>`_, choose the right project, and select your Aiven for ClickHouse service.
+#. In the *Overview* tab, scroll till the section *Service integrations* and click on **Set up integration**. A modal window with available integration options will appear.
+#. Select PostgreSQL from the list of the options.
+#. Choose your service from the list. This is also a place where you can trigger creation of a new Aiven for PostgreSQL service.
+#. Click on **Enable**.
+#. Your PostgreSQL service will be added to the list of enabled service integrations.
+#. You can now close *Service integrations* modal window.
+#. The PostgreSQL integration database will now be added.
+
+The newly created database name has the following format: ``service_PG_SERVICE_NAME_PG_DATABASE_PG_SCHEMA``, for example, ``service_myPGService_myDatabase_mySchema``.
+
+.. note::
+
+    When connecting to an Aiven for PostgreSQL, we connect as the main service user of that service, which has access to all the PostgreSQL tables. SELECT and INSERT privileges are granted to the main service user. It is up to the main service user to grant access to other users.
```

Comment:
should we state here that is `avnadmin`?

Reply:
also, what is this meaning "It is up to the main service user to grant access to other users"?